### DISCUSSION TITLE
pak requirement snippet in wiki does not work copy-pasted asis

### DISCUSSION THREAD

**Discussion by CorradoLanera** [posit-dev/positron#4250] [Feedback] [answered]
`pak` requires confirmation to install packages. So, if the suggested snipped is copy-pasted on an R console and executed, it starts the first dependency search and fails at the first question (for which the copy-pasted queue answers with the second pak call).

![image](https://github.com/user-attachments/assets/19a668ad-df8a-4bcd-a372-35f367b36c00)

Moreover, even if one manually copied-pastes one line at a time (instead of clicking on the copy button of the snippet directly), the number of checks `pak` needs to perform would be greater than necessary, given that most of the suggested packages share common dependencies.

What about leveraging the power of `pak` and suggesting a single call to make it copy-pastable and optimize the dependencies search? 

```r
pak::pak(c(
  "r-lib/usethis",
  "r-lib/cli",
  "r-lib/crayon",
  "r-lib/rlang",
  "r-lib/roxygen2",
  "r-lib/pkgload"
)) 
```

![image](https://github.com/user-attachments/assets/2733c648-de48-4c14-a937-e4e588bd0b28)

**Comment by jennybc** [comment, marked as answer]
I think there was no particular reason to have one `pak::pak()` call per package vs. the `pak::pak(c(...))` syntax, so that's a good move.

I regularly hit return (indicate **Y**es) at the "! pkg is loaded in the current R session ..." decision point. Did that error for you?

Probably most importantly, we no longer need so many dev packages as most of these have had a CRAN release since we first assembled this list and the motivating Positron-favorable changes are in a released version. I know this is true of usethis, for example. It really might be true of *all* of them?

**Comment by CorradoLanera** [comment]
""! pkg is loaded in the current R session ..." decision point. Did that error for you?"
No, I just checked on another package, and it works properly as expected.

![image](https://github.com/user-attachments/assets/b5f88f1b-de3f-488b-b950-ab76e6e29025)

I have no real reason for not doing that; I don't trust myself to remember the defaults 😆 🤣  So, it is faster and safer for me to type what I want explicitly (if I have to hit one answer only) than to check if the default matches my intention and decide whether to go for implicit (saving one type) or explicit. On the other hand, when I have many identical answers (as if I had to copy-paste one `pak` at the time), I go explicit for the first and implicit for the further ones, as you do.
